src/nemo_evaluator/observability/model_traffic.py

```
@@ -75,6 +75,25 @@ def _token_value(value: Any) -> int | None:
         return None
 
 
+def _token_list(value: Any) -> list[int] | None:
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)